### PR TITLE
Install icons, add meson

### PR DESCRIPTION
**Pull request by fufexan**
- **meson: init**
- **nix/package: update derivation**

### CODE REVIEW

**Comment by NotAShelf** [comment]
-3/10 for meson

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,17 @@
+project(
+  'tailray',
+  'rust',
+  version: '0.2.0',
```
Comment (NotAShelf) [inline meson.build]: we're at 0.2.1 now

File: nix/package.nix [hunk 2]
```diff
@@ -1,31 +1,26 @@
...
-      fileset =
-        lib.fileset.intersection
-        (lib.fileset.fromSource (lib.sources.cleanSource ../.))
-        (lib.fileset.unions [
-          ../src
-          ../Cargo.toml
-          ../Cargo.lock
-        ]);
-    };
+    src = ../.;
```
Comment (NotAShelf) [inline nix/package.nix]: why get rid of toSource? this makes the source non-reproducible and puts churn in the store path if I understand source paths correctly

Comment (fufexan) [reply nix/package.nix]: If I leave it there nothing is copied to the build dir.

Comment (NotAShelf) [reply nix/package.nix]: okay I'll change this manually then. until then could you use `builtins.path` ?

**Review by NotAShelf — APPROVED** [review]
I have zero clue how meson works lol

**Pull request merged**